src/integration-test/java/apptest/CaseIT.java

```diff
@@ -0,0 +1,39 @@
+package apptest;
+
+import static org.springframework.http.HttpMethod.GET;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+
+import org.junit.jupiter.api.Test;
+
+import se.sundsvall.alkt.Application;
+import se.sundsvall.dept44.test.AbstractAppTest;
+import se.sundsvall.dept44.test.annotation.wiremock.WireMockAppTestSuite;
+
+@WireMockAppTestSuite(files = "classpath:/Case/", classes = Application.class)
+class CaseIT extends AbstractAppTest {
+
+	private static final String RESPONSE_FILE = "response.json";
+
+	private static final String BASE_URL = "/case/";
+
+	@Test
+	void test01_getCase_shouldReturnCases() {
+		setupCall()
+				.withServicePath(BASE_URL + 2)
```

Comment:
To much indentation?

Reply:
I honestly don't know why it does this..

Reply:
I'm not sure either, but it might have something to do with you pushing code that contains "two tab"-indents?

Reply:
You don't say!
I meant the IDE =P